### DISCUSSION TITLE
https://github.com/PrestaShop/PrestaShop/issues/34605

### DISCUSSION THREAD

**Discussion by TheEnglishGartner** [community/community#122313] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,

Sorry to use a link for subject. But I have the same issue as indicated. You mention that he should disable pm_subscription - could I ask you to elaborate a bit on this? Including where to disable it?

/Lasse

**Comment by ebndev** [comment]
Hi @TheEnglishGartner,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from PrestaShop. You might want to try opening your own issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

**Comment by ebndev** [comment]
Hi again @TheEnglishGartner!

If you don't mind, would you mind letting us know how you initially ended up in the [Profile category](https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+) when creating your discussion and providing the link to the project's repository? We are trying to investigate if there is a bug or link that may be re-directing users from the project's Discussions board, if that’s where you intended to post.

Any insight would be helpful, thanks!

**Reply by ebndev** [reply]
Never mind, found the issue! Thanks again for your time and being a part of the GitHub Community 😄
